static/bidder-info/appstock.yaml

```diff
@@ -0,0 +1,16 @@
+endpoint: "http://ads-pbs.pre.vr-tb.com/openrtb/{{.PublisherID}}?host={{.Host}}"
+maintainer:
+  email: "engineering@project-limelight.com"
```

Comment:
verification sent to this email address. Please reply with confirmed.

Reply:
received